woocommerce/woocommerce-ios
# SPE: Move "Product type" option to product settings

Closes: https://github.com/woocommerce/woocommerce-ios/issues/8838

## Description

This PR adds "Product type" row to the settings screen and hides it from the product details form.

## Testing

1. Build and run the app in debug/alpha mode.
2. Open product details for any product (or start product creation flow).
3. Confirm "Product type" on top level is hidden.
4. Tap "•••" in the navab, then select "Product Settings"
5. Confirm "Product Type" is displayed in the settings.
6. Try updating the product type, save the changes and reopen the product to confirm.

## Screenshots

before|after
--|--
![Simulator Screen Shot - 1](https://user-images.githubusercontent.com/3132438/220061398-e4ca3fc5-7136-422d-b359-aa2ac3136c75.png)|![Simulator Screen Shot - 3](https://user-images.githubusercontent.com/3132438/220062907-bcdee390-e7f7-4c00-a8c6-d47fd281a167.png)
![Simulator Screen Shot - 2](https://user-images.githubusercontent.com/3132438/220061407-24a1cb94-3934-4e3d-b695-c94e95a0b6c6.png)|![Simulator Screen Shot - 4](https://user-images.githubusercontent.com/3132438/220062959-0466926d-372b-4e42-9fbf-eb6fb4ca311e.png)



---
